app/models/user.rb

```diff
@@ -0,0 +1,3 @@
+class User < ActiveRecord::Base
+  has_many  :cards
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.